examples/chat.rs

```diff
@@ -43,6 +43,13 @@ use std::task::{Context, Poll};
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn Error>> {
+    tracing_subscriber::fmt()
+        .with_env_filter(tracing_subscriber::EnvFilter::from_default_env())
+        .with_span_events(tracing_subscriber::fmt::format::FmtSpan::FULL)
+        .with_target(false)
+        .with_timer(())
```

Comment:
Would it be useful to document why these are here, given that this is an example file?

Reply:
Sure, I’ll add some comments. 

Disabling the time stamps and targets was just in order to get shorter log lines so that the example screenshots looked a little nicer; that part would also be fine to just remove.